miranda/convert/data/ecmwf_cf_attrs.json

```diff
@@ -107,6 +107,162 @@
       "standard_name": "snowfall_flux",
       "units": "kg m-2 s-1"
     },
+    "slhf": {
+      "_cf_variable_name": "hfls",
+      "_corrected_units": {
+        "era5-land": "J m-2",
+        "era5-single-levels": "J m-2",
+        "era5-single-levels-preliminary-back-extension": "J m-2"
+      },
+      "_offset_time": {
+        "era5-land": true,
+        "era5-single-levels": true,
+        "era5-single-levels-preliminary-back-extension": true
+      },
+      "_transformation": {
+        "era5-land": "deaccumulate",
+        "era5-single-levels": "amount2rate",
+        "era5-single-levels-preliminary-back-extension": "amount2rate"
+      },
+      "cell_methods": "time: mean (interval: 1 hour)",
+      "comments": "This parameter is the transfer of latent heat (resulting from water phase changes, such as evaporation or condensation) between the Earth's surface and the atmosphere through the effects of turbulent air motion. Evaporation from the Earth's surface represents a transfer of energy from the surface to the atmosphere.",
+      "long_name": "Surface upward latent heat flux",
+      "original_long_name": "Surface latent heat flux",
+      "standard_name": "surface_upward_latent_heat_flux",
+      "units": "W m-2"
+    },
```

Comment:
@tlogan2000 amount2rate and deaccumulation are appropriate here, right?

Reply:
According to @aulemahal, conversion should be supported (though units might be recorded as `J s-1`). Will raise an issue in xclim if errors arise in the conversion.